src/pages/ServiceGraph/SummaryPanel.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import SummaryPanelEdge from './SummaryPanelEdge';
+import SummaryPanelGraph from './SummaryPanelGraph';
+
+type SummaryPanelState = {
+  // stateless
+};
+
+type SummaryPanelProps = {
+  display: string;
+  // data: any;
+};
+
+export default class SummaryPanel extends React.Component<SummaryPanelProps, SummaryPanelState> {
```

Comment:
Instead of re-introducing older version of `SummaryPanel`  as `SummaryPanelGraph` and `SummaryPanelEdge` we could rename `SummaryPanel` to `SummaryPanelHOC` (higher-order component) and in its `render()` produce the appropriate SummaryPanel.

Reply:
if we want the test to pass

```
// App.test.tsx

// comment the following two lines
jest.mock('../../components/SummaryPanel/RpsChart');
jest.mock('../../components/SummaryPanel/ErrorRatePieChart');

// add the following
jest.mock('../../pages/ServiceGraph/SummaryPanelGraph');
jest.mock('../../pages/ServiceGraph/SummaryPanelEdge);
```